easybuild/easyconfigs/g/GROMACS/README.md

```diff
@@ -104,11 +108,19 @@ engines.
     of enabling support for the `cray-python` modules. Therefore the corresponding 
     EasyConfigs of GROMACS have also been replaced.**
 
+### GROMACS 2023.2 with AMD GPU support for CPE 22.12
+
+  * There are two easyconfig files for running GROMACS with AMD GPU acceleration (ROCm support via hipSYCL), that follows [installation guide](https://manual.gromacs.org/2023.2/install-guide/index.html#sycl-gpu-acceleration-for-amd-gpus):
+
+    * MPI version for multi-node use, with [HeFFTe](https://manual.gromacs.org/2023.2/install-guide/index.html#using-heffte) supporting offload to multiple GPUs 
+    ([GPU aware MPI](https://manual.gromacs.org/2023.2/install-guide/index.html#gpu-aware-mpi-support) enabled, support for PME decomposition)
```

Comment:
```suggestion
    * MPI version for multi-node use, with [HeFFTe](https://manual.gromacs.org/2023.2/install-guide/index.html#using-heffte) supporting offload to multiple GPUs 
    ([GPU aware MPI](https://manual.gromacs.org/2023.2/install-guide/index.html#gpu-aware-mpi-support) enabled, support for GPU PME decomposition)
```

Reply:
I have rewritten these points in the meantime following your previous suggestion:
* MPI versions are recommended to use on LUMI
   * [HeFFTe](https://manual.gromacs.org/2023.2/install-guide/index.html#using-heffte) variant allows offload to multiple GPUs (relies on rocFFT)
   * [VkFFT](https://manual.gromacs.org/2023.2/install-guide/index.html#using-vkfft) variant is faster but does not support PME decomposition, recommended for a single GPU runs (standalone or ensemble) or in multi GPU runs with exactly one separate PME rank (i.e. `-npme 1` runtime option).
* thread-MPI is for a single node use only and does not support direct GPU communication, recommended only for a single GPU use.